WooCommerce/Classes/ViewRelated/JetpackSetup/AdminRole/AdminRoleRequiredViewModel.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import Yosemite
+
+/// View model for `AdminRoleRequiredView`
+///
+final class AdminRoleRequiredViewModel {
+    let username: String
+    let roleName: String
+
+    private let siteID: Int64
+
+    private let stores: StoresManager
+    init(siteID: Int64, stores: StoresManager = ServiceLocator.stores) {
+        self.siteID = siteID
+        self.stores = stores
+        self.username = stores.sessionManager.defaultCredentials?.username ?? ""
+        self.roleName = stores.sessionManager.defaultRoles
+            .map { $0.displayString() }
+            .joined(separator: ", ")
+    }
+
+    @MainActor
+    func reloadRoles() async throws -> Bool {
```

Comment:
Nit: The name `reloadRoles()` doesn't say anything about the return value `Bool`. Kindly consider renaming.

Reply:
Good point - renamed in 46467939fdbeddb7262e5dc94d946ce84ce54dac.